crates/oxc_linter/src/snapshots/eslint_no_control_regex@capture-group-indexing.snap

```diff
@@ -6,25 +6,25 @@ source: crates/oxc_linter/src/tester.rs
  1 │ const r = /\0/;
    ·           ────
    ╰────
-  help: '\0' is not a valid control character.
+  help: '\0' matches the null character (U+0000), which is a control character. If you intend to match the null character, consider using '\x00' or '\u0000' for clarity.
```

Comment:
This suggestion doesn't seem correct. Both `\x00` or `\u0000` will be warned by this rule.
[playground](https://playground.oxc.rs/#eNpVkE1rwzAMhv9K0bmsaWGXjN1GrhvbNTAURykejh0kuWsp+e9TEpJRXyyhj+d9dQcHJbgURXffA6oSx+PudXeo62tRHF7q+Fg7LbVc2JuqsIcE5R04x+mTW1S8QqmcaQ/BR11jcWmgLbn1TQprpoxRusQ9lB0GoXEPA7IQTxsxhPT7SZo5vmcV31KVo1OfjLeMD0zWe6EPZIqyIVJmR5UPFLE3MCiJPv0I2PZJ17zd4g2+4BT5TCYaSE7F8dnseUkBldo3cgEZJ7JBVqEutXSm2buBmkBfM7fH4b+n99F3fgXaPZVTqMzWNHUhbpKYwKV7HP8AO4iN7A==)

I kind of opened the issue straight ahead without thinking because `\0` was not warned by eslint. But on second thought, given that `\0` is a control character, it actually makes sense for this rule to catch it.

Reply:
https://github.com/oxc-project/oxc/pull/10993